KoreBuild-dotnet/build/dotnet-install.ps1

```diff
@@ -59,6 +59,8 @@ if (Test-Path $LocalFile)
                 say "You already have the latest version"
                 exit 0
             }
+            $DotNetFileName="dotnet-win-x64.$RemoteVersion.zip"
```

Comment:
Why does, `dotnet-win-x64.latest.zip` not work? We shouldn't need to specify the Latest version: https://github.com/aspnet/Universe/pull/383#discussion_r52815601